src/celeritas/ext/GeantImporter.cc

```diff
@@ -192,6 +192,72 @@ struct MatPropGetter
     }
 };
 
+//---------------------------------------------------------------------------//
+//! Map particles defined in \c G4MaterialConstPropertyIndex .
+std::map<std::string, PDGNumber> optical_particles_map()
+{
+    return {{"PROTON", pdg::proton()},
+            {"DEUTERON", pdg::deuteron()},
```

Comment:
Are these PDGs used anywhere?

Reply:
Yes. Even though they are stored as `int`, we use those to construct a `map<pdg, spectrum>`. Although not in this PR, these pdgs are then used in the `ScintillationParams` to store particle spectrum according to particle id in Celeritas.

Reply:
Hmm I'm still not seeing where that's done... I see the map keys being used, but not the values anywhere. I'm also not seeing where the `particles` map in `ImportScintData` is saved... it looks like the particle scintillation spectra are created but never actually stored in the imported data?

Reply:
Ohh, thanks! Great catch. I mistakenly deleted the `insert()` call to  `ImportScintData::particles` while moving code around.